src/components/filterByDuration/ButtonFormField.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function ButtonFormField() {
+  return (
+    <div className="form-group">
+      <Link to="/" className="btn">
```

Comment:
Please add a title attribute on this button, to improve accessibility

Reply:
Resolve in commit 24dc47fd5508dbeec763700c20b1df2b75b8a31c
Adding a title to the button in the ButtonFromField.jsx file